ui/src/components/icon/OutboundCitationIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+interface Props {
+    width: string;
+    height: string;
+}
+
+export const OutboundCitationIcon = ({ width, height }: Props) => (
+  <svg width={width} height={height} viewBox="0 0 27 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It's worth adding some kind of identifier to this element so when you look it up in the DOM inspector, you know which icon it is. Maybe add a `data-icon="outbound-citation"`?